### PR TITLE
add `MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT`

### PR DESCRIPTION
**Pull request by realbigsean**

### CODE REVIEW

**Review by tersec — APPROVED** [review]

File: custom_config_data/config.yaml [hunk 1]
```diff
@@ -61,6 +61,8 @@ INACTIVITY_SCORE_RECOVERY_RATE: 16
 EJECTION_BALANCE: 28000000000
 # 2**2 (= 4)
 MIN_PER_EPOCH_CHURN_LIMIT: 4
+# 2**3 (= 8)
+MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT: 8
 # 2**16 (= 65,536)
 CHURN_LIMIT_QUOTIENT: 65536
```
Comment (etan-status) [inline custom_config_data/config.yaml:67]:
```suggestion
# 2**16 (= 65,536)
CHURN_LIMIT_QUOTIENT: 65536
# [New in Deneb:EIP7514] 2**3 (= 8)
MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT: 8
```

**Comment by tersec** [comment]
https://github.com/ethereum/consensus-specs/blob/7402712e4faebcb9cd87370d91acf638ae168949/configs/mainnet.yaml#L78-L91 lists `CHURN_LIMIT_QUOTIENT` before `MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT` while this puts them in the opposite order. Consistency with the spec might be useful to more readily verify matching.

File: custom_config_data/config.yaml (hunk 1 above)
Comment (etan-status) [reply custom_config_data/config.yaml:67]: note, this simply is the default value from https://github.com/ethereum/consensus-specs/blob/v1.4.0-beta.6/configs/mainnet.yaml

so, technically it is correct even when omitted, as this config is based on `PRESET_BASE = 'mainnet'`

**Comment by realbigsean** [comment]
whoops! thanks, should be set now

**Review by g11tech — APPROVED** [review]
lgtm

**Comment by parithosh** [comment]
Do we want to set the value at 8 or higher? Considering that Holesky has a larger validator set to begin with

**Comment by barnabasbusa** [comment]
I'd vote for sticking with mainnet defaults. There is not much use for having higher than 8 deposits ever processed.

**Comment by g11tech** [comment]
> I'd vote for sticking with mainnet defaults. There is not much use for having higher than 8 deposits ever processed.

+1

**Review by parithosh — APPROVED** [review]

**Review by pk910 — APPROVED** [review]

**Pull request merged**